components/BottomAppBar/README.md

```diff
@@ -115,3 +119,71 @@ tracking its development:
 - Task: [Implement a color themer](https://www.pivotaltracker.com/story/show/157095394)
 - Task: [Impement a surface variant themer](https://www.pivotaltracker.com/story/show/156933141)
 
+
+## Accessibility
+
+<!-- Extracted from docs/accessibility.md -->
+
+To help ensure your bottom app bar is accessible to as many users as possible, please be sure to review the
+following recommendations:
+
+### Set `-accessibilityLabel`
+
+Set an appropriate
+[`accessibilityLabel`](https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619577-accessibilitylabel)
+to all buttons within the bottom app bar.
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+
+```swift
+bottomAppBar.floatingButton.accessibilityLabel = "Compose"
+let trailingButton = UIBarButtonItem()
+trailingButton.accessibilityLabel = "New"
+bottomAppBar.trailingBarButtonItems = [ trailingButton ]
+```
+
+#### Objective-C
+
+```objc
+bottomAppBar.floatingButton.accessibilityLabel = @"Compose";
+UIBarButtonItem *trailingButton = 
+    [[UIBarButtonItem alloc] initWithTitle:nil
+                                     style:UIBarButtonItemStylePlain
+                                    target:self
+                                    action:@selector(didTapTrailing:)];
+trailingButton.accessibilityLabel = @"Buy";
```

Comment:
Super-nit: Can we pick "New" or "Buy" and use it for both examples?

Reply:
Done